packages/features/ee/workflows/lib/reminders/emailReminderManager.ts

```diff
@@ -377,6 +378,8 @@ export const deleteScheduledEmailReminder = async (reminderId: number, reference
         cancelled: true,
       },
     });
+
+    await cancelScheduledEmailSend(referenceId);
```

Comment:
We need to do this in the cron job, there is a limit of canceled emails in the scheduled_sends queue which we will reach otherwise

Reply:
Got it ! Will remove this ..